vars/common.yml

```diff
@@ -9,18 +9,24 @@ search-api:
 
 admin-frontend:
   subdomain: dm
-  path: /admin
+  paths:
+    - /admin
 
 buyer-frontend:
   subdomain: dm
+  paths:
+    - ''
+    - /buyers/direct-award
 
 supplier-frontend:
   subdomain: dm
-  path: /suppliers
+  paths:
+    - /suppliers
 
 briefs-frontend:
   subdomain: dm
-  path: /buyers
+  paths:
+    - /buyers
 
 brief-responses-frontend:
```

Comment:
I think the path for the brief-responses app needs moving to the array pattern like the others.